backend/src/test/java/gov/cdc/usds/simplereport/api/converter/FhirConverterTest.java

```diff
@@ -1244,7 +1245,7 @@ void createFhirBundle_TestEvent_matchesJson() throws IOException {
     ReflectionTestUtils.setField(
         person, "phoneNumbers", List.of(new PhoneNumber(PhoneType.LANDLINE, "7735551234")));
 
-    var actual = createFhirBundle(testEvent, gitProperties, date);
+    var actual = createFhirBundle(testEvent, gitProperties, date, "P");
 
     String actualSerialized = parser.encodeResourceToString(actual);
 
```

Comment:
It seems that `strict=false` on line 1279 is way too lenient. Can you change the last parameter to `JSONCompareMode.NON_EXTENSIBLE`. This will also make the bundle.json fail comparison which you will need to add the meta to.

Reply:
Thanks, I was wondering why that was passing, should be fixed now, I added the JSONCompareMode.NON_EXTENSIBLE made sure the tests failed, then updated the bundle with the meta tag and that made it pass